docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+
+This is an overview of Fleet SDK. See [Getting Started](https://fleet-sdk.github.io/docs/guide/getting-started) if 
+you want to dive right in.
+
+## What is Fleet SDK? 
+
+Fleet SDK (pronounced /flēt/) is a TypeScript framework for building simple or complex applications on the [Ergo 
+blockchain](https://ergoplatform.org/). It builds on top of [Sigma.js](https://github.
+com/ScorexFoundation/sigmastate-interpreter) and provides a declarative and component-based model to
+communicate with Ergo nodes, the ErgoScript interpreter and Ergo protocol implementations.
```

Comment:
1. Actually Fleet is an off-chain framework
2. Fleet it not entirely build on top of Sigma.JS, only the interpreter and compiler packages are, see: https://github.com/fleet-sdk/fleet#packages
3. Fleet is, in its conception, an off line library, so I think we can remove "communicate with Ergo nodes" part

Maybe worths writing something along these lines: "Feet SDK is an easy-to-use, modular and extensible Ergo Platform's off-chain code SDK (Software Development Kit), written entirely in TypeScript." as in "Getting Started"

Reply:
Yeah, I'm trying to get my head around it, and much of this is new to me. Appreciate any direction when I'm wrong. I apologize for here in the beginning. I might slow you down at first while I get up to speed.

Reply:
Don't worry about that, my friend )